docs/indepth/installation.rst

```diff
@@ -437,70 +476,41 @@ The optional ``port`` variable is the port on the control node to connect to:
    "control-service":
       "hostname": "${CONTROL_NODE}"
       "port": 4524
-  	"dataset":
-  	  "backend": "zfs"
-
-.. task:: enable_flocker_agent ubuntu-14.04 ${CONTROL_NODE}
-   :prompt: [root@agent-node]#
-
-What to do next
----------------
-
-You have now installed ``clusterhq-flocker-node`` and created a ZFS for it.
-
-Next you may want to perform the steps in :doc:`the tutorial <./tutorial/moving-applications>`, to ensure that your nodes are correctly configured.
-Replace the IP addresses in the ``deployment.yaml`` files with the IP address of your own nodes.
-Keep in mind that the tutorial was designed with local virtual machines in mind, and results in an insecure environment.
-
-
-ZFS Backend Configuration
--------------------------
-
-The ZFS backend requires ZFS to be installed.
-
-
-Installing ZFS on CentOS 7
-..........................
-
-Installing ZFS requires the kernel development headers for the running kernel.
-Since CentOS doesn't provide easy access to old package versions,
-the easiest way to get appropriate headers is to upgrade the kernel and install the headers.
-
-.. task:: upgrade_kernel centos-7
-   :prompt: [root@centos-7]#
+   "dataset":
+      "backend": "zfs"
+      "pool": "flocker"
```

Comment:
This has to match the pool created in `task_create_flocker_pool_file` right? If so, document this.